lib/mwlib/src/MW/Media/Image/Imagick.php

```diff
@@ -157,29 +157,37 @@ public function save( string $filename = null, string $mimetype = null ) : ?stri
 	 *
 	 * @param int|null $width New width of the image or null for automatic calculation
 	 * @param int|null $height New height of the image or null for automatic calculation
-	 * @param bool $fit True to keep the width/height ratio of the image
+	 * @param int|null $fit 0 keeps image ratio, 1 enforces target size with scaling, 2 with cropping
 	 * @return \Aimeos\MW\Media\Image\Iface Self object for method chaining
 	 */
-	public function scale( int $width = null, int $height = null, bool $fit = true ) : \Aimeos\MW\Media\Image\Iface
+	public function scale( int $width = null, int $height = null, int $fit = null ) : \Aimeos\MW\Media\Image\Iface
```

Comment:
Why did you use `int|null` for the `$fit` parameter? NULL is no used value.

Reply:
You're right, I used it in a previous version. Cleaned it up.